api/queries_repo.go

```diff
@@ -524,6 +525,35 @@ func ForkRepo(client *Client, repo ghrepo.Interface, org string) (*Repository, e
 	}, nil
 }
 
+func LastCommit(client *Client, repo ghrepo.Interface) (*git.Commit, error) {
```

Comment:
It's unfortunate that now the `api` package has to import (and thus depend on) the whole `git` package just so it can return a `git.Commit` type. Ideally `api` and `git` packages should be entirely separated concerns, unaware of each others' existence. I have some ideas how to iron this out, but I don't think it's time yet, and since this isn't a pressing issue, let's leave this like this and circle back later.